easybuild/easyconfigs/g/gensim/gensim-0.13.2-foss-2016a-Python-2.7.11.eb

```diff
@@ -0,0 +1,43 @@
+easyblock = 'Bundle'
+
+name = 'gensim'
+version = '0.13.2'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://pypi.python.org/pypi/gensim'
+description = """Gensim is a Python library for topic modelling, document indexing and similarity retrieval with
+ large corpora."""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+# this is a bundle of Python packages
+exts_defaultclass = 'PythonPackage'
+
+dependencies = [
+    ('Python', '2.7.11'),
+    ('requests', '2.10.0', '-Python-%(pyver)s'),
+]
+
+
+exts_list = [
+    ('boto', '2.42.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/b/%(name)s'],
+        'source_tmpl': '%(name)s-%(version)s.tar.gz',
+    }),
+    ('bz2file', '0.98', {
+        'source_urls': ['https://pypi.python.org/packages/source/b/%(name)s'],
+        'source_tmpl': '%(name)s-%(version)s.tar.gz',
+    }),
+    ('smart_open', '1.3.4', {
+        'source_urls': ['https://pypi.python.org/packages/source/s/%(name)s'],
+        'source_tmpl': '%(name)s-%(version)s.tar.gz',
+    }),
+    (name, version, {
+        'source_urls': ['https://pypi.python.org/packages/source/m/matplotlib'],
+    }),
+]
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
```

Comment:
@RvDijk `ERROR: Failed to get application instance for gensim (easyblock: Bundle): List of sources for bundle itself must be empty, found ['gensim-0.13.2.tar.gz']`